src/components/ChatAsideProfile/index.ts

```diff
@@ -0,0 +1,17 @@
+import Block from '../../utils/Block';
+import template from './template';
+import styles from './style.module.less';
+
+interface ChatAsideProfileProps {
+  events?: Record<string, never>
```

Comment:
здесь лучше использовать [union type](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-type)
```suggestion
  events?: 'click' | 'hover' | и т.д.
```

Reply:
Привет! Спасибо за ревью, для меня это очень важно. TypeScript у меня откровенно слабоват, хотя я и JS никгда специально не изучал. Постараюсь это наверстать. 

Мне этот интерфейс тут вообщем то и не нужен, оставил его что бы была единая структура. По сути мне тут нужен пустой интерфейс ChatAsideProfileProps = {}, но по условиям спринта так делать нельзя.
заменил на 
events?: never; 

Наверное будет правильно создать новый тип, что то типа ComponentEventsType и туда через union добавлять все допустимые ивенты

type ComponentEventsType = 'click' | 'hover' | .. | never;
и потом уже в интерфейсе events?: ComponentEventsType;

Reply:
только never тут будет лишним, потому что и без never компилятор выдаст ошибку, если что-то другое передать. Never полезен, если нужно ограничить свойства объекта или реализовать [exhaustive switch](https://medium.com/technogise/type-safe-and-exhaustive-switch-statements-aka-pattern-matching-in-typescript-e3febd433a7a)